src/Output/Tty.php

```diff
@@ -20,19 +17,11 @@ class Tty extends Simple
     /** @var TestOutput[] */
     private $testOutputs = [];
 
-    public function __construct(LoopInterface $loop)
+    public function __construct()
     {
         parent::__construct();
 
-        $this->loop = $loop;
         $this->setTerminalSize();
-
-        if (function_exists('pcntl_signal')) {
-            $pcntl = new PCNTL($loop);
-            $pcntl->on(SIGWINCH, function () {
-                $this->setTerminalSize();
-            });
-        }
```

Comment:
What's the replacement for that?

Reply:
Forget about that, Loop::onSignal should do the job

Reply:
Yes, but you added that nowhere, or did I miss it?